Open Judge System/Workers/OJS.Workers.ExecutionStrategies/RubyExecutionStrategy.cs

```diff
@@ -0,0 +1,68 @@
+﻿namespace OJS.Workers.ExecutionStrategies
+{
+    using System.IO;
+
+    using OJS.Common.Extensions;
+    using OJS.Workers.Checkers;
+    using OJS.Workers.Common;
+    using OJS.Workers.Executors;
+
+    public class RubyExecutionStrategy : ExecutionStrategy
+    {
+        protected const string SubmissionFileName = "_$Submission.rb";
+
+        public RubyExecutionStrategy(string rubyPath)
+        {
+            this.RubyPath = rubyPath;
+            this.WorkingDirectory = DirectoryHelpers.CreateTempDirectory();
```

Comment:
You don't need a temp directory, as you only need a single temporary file to keep the code while it is executed.